src/heap_ptmalloc.cpp

```diff
@@ -2089,6 +2103,25 @@ static bool get_glibc_version(void)
 {
 	const size_t bufsz = 64;
 	char buf[bufsz];
+#ifdef FIXME_LIBC_VERSION
+	struct symbol *sym;
+	struct value *val;
+	char* data;
+	/*
+	 * Global var
+	 * File malloc.c: static struct malloc_par mp_;
+	 */
+	sym = lookup_symbol("__libc_version", 0, VAR_DOMAIN, 0).symbol;
+	if (sym == NULL) {
+		CA_PRINT("Failed to lookup gv \"mp_\"\n");
+		return false;
+	}
+	val = value_of_variable(sym, 0);
+	data = (char*)value_as_address(val);
+	
+	const char* version = data;
+
+#endif // Fixme: We should read the libc_version from the debugee, but the above code never works.
```

Comment:
This is a static var. Maybe `VAR_DOMAIN` is not enough.

Reply:
I tried LABEL_DOMAIN, but it is not work either. I cannot find a suitable domain from
```c

typedef enum domain_enum_tag
{
  /* UNDEF_DOMAIN is used when a domain has not been discovered or
     none of the following apply.  This usually indicates an error either
     in the symbol information or in gdb's handling of symbols.  */

  UNDEF_DOMAIN,

  /* VAR_DOMAIN is the usual domain.  In C, this contains variables,
     function names, typedef names and enum type values.  */

  VAR_DOMAIN,

  /* STRUCT_DOMAIN is used in C to hold struct, union and enum type names.
     Thus, if `struct foo' is used in a C program, it produces a symbol named
     `foo' in the STRUCT_DOMAIN.  */

  STRUCT_DOMAIN,

  /* MODULE_DOMAIN is used in Fortran to hold module type names.  */

  MODULE_DOMAIN,

  /* LABEL_DOMAIN may be used for names of labels (for gotos).  */

  LABEL_DOMAIN,

  /* Fortran common blocks.  Their naming must be separate from VAR_DOMAIN.
     They also always use LOC_COMMON_BLOCK.  */
  COMMON_BLOCK_DOMAIN,

  /* This must remain last.  */
  NR_DOMAINS
} domain_enum;

```

Reply:
solved in https://github.com/yanqi27/core_analyzer/pull/38/files